src/common/storage/src/config.rs

```diff
@@ -407,8 +416,57 @@ impl Debug for StorageOssConfig {
 }
 
 /// config for Moka Object Storage Service
-#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
-pub struct StorageMokaConfig {}
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct StorageMokaConfig {
```

Comment:
If the cache backend as a Storage config, how can I make the cache to the local disk?

Reply:
From the databend view, prefer to add a Cache config under the storage config item? Then we can set CACHE_TYPE which we want, this looks more clear

Reply:
> If the cache backend as a Storage config, how can I make the cache to the local disk?

Use `CACHE_TPYE=fs`.

> From the databend view, prefer to add a Cache config under the storage config item? Then we can set CACHE_TYPE which we want, this looks more clear

Yes, this is exactly how we implement cache. We just reuse the same storage config.

Reply:
From the design, Cache is a part of the storage(hot and fast layer), is it better if we move the cache config under the storage item?

Reply:
> is it better if we move the cache config under the storage item?

Ideally, we should have the following structure:

```toml
[storage]

[storage.data]
type = "s3"

[storage.cache]
type = "redis"

[storage.temperary]
type = "s3"
```

But for historical reasons, the `storage` here is occupied by `storage.data`. So we must add a new `cache` section along with `storage`.

Maybe we can do this refactor?

Reply:
I like the idea. Since the config is defined, the user will use it, and it's hard to change it later :)